apps/aecore/test/aec_block_micro_candidate_tests.erl

```diff
@@ -180,16 +195,358 @@ block_extension_test_() ->
         end}
       ]}.
 
+used_gas_test_() ->
+    {foreach,
+      fun() ->
+        meck:new(aeu_time, [passthrough]),
+        meck:new(aec_chain, [passthrough]),
+        meck:new(aec_db, [passthrough]),
+        meck:new(aec_keys, [passthrough]),
+        meck:new(aec_tx_pool, [passthrough]),
+        meck:new(aec_trees, [passthrough])
+      end,
+      fun(_) ->
+        meck:unload(aec_trees),
+        meck:unload(aec_tx_pool),
+        meck:unload(aec_keys),
+        meck:unload(aec_chain),
+        meck:unload(aec_db),
+        meck:unload(aeu_time)
+      end,
+      [{"Check consumed gas by a contract create tx",
+        fun() ->
+          Height = 10,
+          Protocol = aec_hard_forks:protocol_effective_at_height(Height),
+          AccMap = #{ preset_accounts => [{?TEST_PUB, 1000000000000000000000000000000000000000}] },
+          {Block00, Trees0} = aec_block_genesis:genesis_block_with_state(AccMap),
+          %% since the block is genesis one, it does not have the protocol
+          %% enabled, so we set it
+          Block0 = set_protocol_and_version(Block00, Protocol, Height),
+          ContractName = identity,
+          VM = 
+              case Protocol >= ?IRIS_PROTOCOL_VSN of
```

Comment:
Lima should prefer FATE? I don't think there was a single AEVM contract created on Lima